getting-started/bookshelf/config.go

```diff
@@ -37,84 +41,99 @@ var (
 	_ mgo.Session
 )
 
-const PubsubTopicID = "fill-book-details"
-
 func init() {
 	var err error
 
+	// Read config.json
+	configFile, err := ioutil.ReadFile("config.json")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Parse config.json
+	var configJson map[string]interface{}
```

Comment:
All of the values are strings. So, this can be `map[string]string`.

Alternatively, make it a `struct` with all of the fields defined, so it's a bit easier to use.